README.md

```diff
@@ -183,9 +183,25 @@ optional arguments:
   --infected INFECTED   Number of infected people in each region/age group
                         (default: 100)
 ```
- 
+
 Descriptions of the data files used can be found in the [data dictionary](sample_input_files/README.md).
 
+## Building the documentation
+
+To build a local version of the readthedocs site e.g. when working on the documentation.
```

Comment:
Perhaps one could rephrase this to avoid terminating an incomplete sentence with a full stop.